src/main/java/org/gridsuite/modification/server/service/NetworkStoreListener.java

```diff
@@ -73,7 +73,7 @@ public static NetworkStoreListener create(Network network, UUID networkUuid, UUI
                                               EquipmentInfosService equipmentInfosService,
                                               boolean isBuild, boolean isApplyModifications) {
         var listener = new NetworkStoreListener(network, networkUuid, groupUuid, modificationRepository, equipmentInfosService,
-            isBuild, isApplyModifications);
+                isBuild, isApplyModifications);
```

Comment:
All changes in this source seem to be indentation changes.

Reply:
my bad